### ISSUE TITLE
sudo bash ee sent installer to background

### ISSUE THREAD

**Issue by rippler** [rtCamp/easyengine#476]
1. install clean Ubuntu 14.04
2. add a non-root user
3. as this new user, run `wget -qO ee rt.cx/ee && sudo bash ee`

I get that `Executing apt-get update, please wait...` message and 5 minutes later nothing happens.

Then I open another console as root, and run:
`tail -f /var/log/ee/install.log`
I get:
```
root@backup:~# tail -f /var/log/ee/install.log
Setting up python3-pkg-resources (3.3-1ubuntu1) ...
Setting up python3-setuptools (3.3-1ubuntu1) ...
Setting up sqlite3 (3.8.2-1ubuntu2) ...
Processing triggers for libc-bin (2.19-0ubuntu6.4) ...
Cloning EasyEngine, please wait...
Installing EasyEngine, please wait...
EasyEngine (ee) required your name & email address to track changes you made under the Git version control
EasyEngine (ee) will be able to send you daily reports & alerts in upcoming version
EasyEngine (ee) will NEVER send your information across
Enter your name: Name not Valid, Please enter again
Enter your name: 
```
Funny enough, I entered my name in this output of `tail` and it looks like it prompted me to enter it again. Or maybe the first console reacted to a key I may have pressed when nothing was happening, but in any case, I didn't see the name prompt in console where I was installing `ee`.

**Comment by rippler** [comment]
What appears to fix it:
`apt-get update ; apt-get upgrade -y`
`apt-get install libiconv-hook-dev libiconv-hook1`
No reboot is required. Then run the installer. 
I don't know if all the steps above are required, but that's what works.

**Comment by harshadyeola** [comment]
@rippler 
This is My Fresh Vagrant Box
``` 
vagrant@ubuntu-web:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.1 LTS
Release:	14.04
Codename:	trusty
```

```vagrant@ubuntu-web:~$ wget -qO ee rt.cx/ee && sudo bash ee
Executing apt-get update, please wait...
Creating EasyEngine log directory, please wait...
Installing required packages, please wait...
Reading package lists...
Building dependency tree...
Reading state information...
gcc is already the newest version.
gcc set to manually installed.
gzip is already the newest version.
python3 is already the newest version.
.
.
.
.
Setting up sqlite3 (3.8.2-1ubuntu2) ...
Processing triggers for libc-bin (2.19-0ubuntu6.4) ...
Cloning EasyEngine, please wait...
Installing EasyEngine, please wait...
EasyEngine (ee) required your name & email address to track changes you made under the Git version control
EasyEngine (ee) will be able to send you daily reports & alerts in upcoming version
EasyEngine (ee) will NEVER send your information across
Enter your name: 
```

**Comment by rippler** [comment]
Well, mine is a fresh Digital Ocean 14.04 x64 box, installed it 30 mins ago. 
If you by default had the 2 packages that I installed, then your example doesn't count ;-)

**Comment by harshadyeola** [comment]
@rippler 

I don't have these packages in my system
```
root@ubuntu-web:~# dpkg -l | grep libiconv-hook-dev
root@ubuntu-web:~# dpkg -l | grep libiconv-hook1
root@ubuntu-web:~# 
```

**Comment by harshadyeola** [comment]
@rippler 
Well thanks for update, will check on fresh DO instance.

**Comment by rippler** [comment]
Aah, then that's different :-)
I want to contribute back so I destroyed the first one and created another one.
It seems Digital Ocean provisioning system is having issues. I didn't try to look at the console the first time around, but this time I did and once it came up, it threw this error:
![screenhunter_35 mar 26 16 40](https://cloud.githubusercontent.com/assets/4045519/6842817/30990cfc-d3d7-11e4-86d8-1054a0d39da1.jpg)

However, this particular instance worked. I can only speculate that earlier provisioning didn't work as expected. 
So I decided to destroy that one and create another one :-)
I again saw the same error about missing final cloud modules, and it again worked.

I was using the same everything in each attempt:
* 512 MB
* NYC2
* Ubuntu 14.04 x64
* I even got the same IP every time as I was recycling them

I don't know what happened, but it was real (I tried several times and watched it with strace). Thanks for your time!
